docs/hardware/Project Babble/babble-omnicept.mdx

```diff
@@ -0,0 +1,142 @@
+---
+sidebar_position: 2
+sidebar_label: Project Babble - HP Reverb G2 Omnicept Setup Guide
+---
+
+# Project Babble - HP Reverb G2 Omnicept Setup Guide
+
+## Summary
+
+The [HP Reverb G2 Omnicept Edition](https://www.hp.com/us-en/vr/reverb-g2-vr-headset-omnicept-edition.html) is a VR headset with eye tracking and a lower face camera. While there is a module that allows you to use the headset's eye tracking with VRCFaceTracking, the face camera by itself does not do anything.
+
+<div style={{
+    width: '50%',
+    height: 'auto',
+    margin: 'auto',
+    display: 'block'
+}}>
+    <img src={require("../img/babble/hp_face_cams_RTV.jpg").default} alt="HP Reverb G2 Omnicept eye and face cameras, circled in red" />
+</div>
+
+***But!*** With Project Babble this camera can be used to track your facial expressions! This guide will show you how to use the lower face camera with VRCFaceTracking.
+
+## Setup
+
+You will need the following software installed to use the lower face camera with VRCFaceTracking:
+- The [VRCFT-Babble Module](./babble-setup)
+- [OBS](https://obsproject.com/download)
+- [Project Babble](https://github.com/SummerSigh/ProjectBabble/releases/latest)
+- The [Glia Runtime/SDK](https://developers.hp.com/omnicept/downloads)
+- The [Glia Lip Viewer](https://discord.com/channels/849300336128032789/915075185328152606/1122604182130675772)
```

Comment:
Consider adding a warning that you must be a member of the discord to access this using `<CustomLink to="discord"/>`

Reply:
Replaced with an absolute link to `GliaLipViewer.zip` in [#7e8217c](https://github.com/VRCFaceTracking/docs/pull/15/commits/7e8217c2b17f53f736a68f24dedd74791c549a52)